browser/src/Editor/NeovimEditor/NeovimEditor.tsx

```diff
@@ -714,6 +714,7 @@ export class NeovimEditor extends Editor implements IEditor {
             )
         }
 
+        this._workspace.autoDetectWorkspace()
```

Comment:
Would it make sense for us to check this on buffer enter? I'm thinking of the case where I open Oni, with no workspace open, and then open a file - do you think we should try and `auto-detect` the workspace at that time?